pkg/ui/src/views/cluster/containers/nodeGraphs/summaryBar.tsx

```diff
@@ -86,11 +93,17 @@ export default function(props: ClusterSummaryProps) {
                                         usable storage capacity across all nodes.`} />
         </SummaryStat>
         <SummaryStat title="Unavailable ranges" value={props.nodesSummary.nodeSums.unavailableRanges} />
-        <SummaryMetricStat id="qps" title="Queries per second" format={formatOnePlace} >
-          <Metric sources={props.nodeSources} name="cr.node.sql.query.count" title="Queries/Sec" nonNegativeRate />
-        </SummaryMetricStat>
-        <SummaryMetricStat id="p50" title="P50 latency" format={formatNanosAsMillis} >
-          <Metric sources={props.nodeSources} name="cr.node.sql.service.latency-p50" aggregateMax downsampleMax />
+        <SummaryMetricStat
+          id="qps"
+          title="Queries per second"
+          format={formatOnePlace}
+          aggregator={TimeSeriesQueryAggregator.SUM}
+          summaryStatMessage="Sum of Selects, Updates, Inserts, and Deletes across your entire cluster."
+        >
+          <Metric sources={props.nodeSources} name="cr.node.sql.select.count" title="Queries/Sec" nonNegativeRate />
```

Comment:
Using these pseudo-component things to specify metrics to be summed is so bizarre. But, changing that is out of scope for this PR.